docs/_getting_started/jupyter.md

```diff
@@ -114,11 +246,11 @@ For example, try to navigate to `intro_yens_sep_2023` folder in file brower firs
 
```

Comment:
>For example, try to navigate to `intro_yens_sep_2023` folder in file brower first then double click on `investment-npv-parallel.R` file to open it in the text editor:

This doesn't make sense out of context -- also, the image is broken

Reply:
deleted and fixed